css/homestyle.css

```diff
@@ -0,0 +1,123 @@
+body, html {
+    padding: 0;
+    margin: 0;
+    height: 100%;
+}
+.coverImage {
+    background-color: #000;
+    background: url("../Photos/hero-image 2.png") no-repeat center center fixed; 
+    background-size: cover;
+    position: relative;
+    width: 100%;
+    height: 100%;
+}
+.coverContainer {
+    width: 100%;
+    height: 100%;
+}
+.sidebar {
+    background-color:#FFFFFF;
+    top:0;
+    bottom:0;
+    position: absolute;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+.text {
+    font-family: Sofia Pro;
+    font-size:14px;
+    line-height:22px;
+    letter-spacing:2px;
+    padding-right: 10px;
+    padding-left:10px;
+    writing-mode: vertical-rl;
+    transform: rotate(180deg);
+}
+.dot {
+    padding-top: 25px;
+    padding-bottom: 25px;
+}
+.subtitle {   
+    font-family: CircularStd;
+    font-size:16px;
+    line-height: 19px;
+    color:#FFFFFF;
+    letter-spacing: 2px;    
+}
+.title {
+    font-family: DM Serif Display;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 130px;
+    letter-spacing: 19px;
+    color: #FFFFFF;
+}
+.description { 
+    
+    color: #FFFFFF;
+    font-family: Sofia Pro;
+    font-size:18px;
+    line-height:29px;
+    letter-spacing: 1.72px;;
+}
+.shape {
+    position:absolute;
+    left:15%;
+    right:73.12%;
+    top:3.51%;
+    bottom:94.53%;
+}
+.mention { 
+    color: #FFFFFF;
+    font-family: Sofia Pro;
+    font-size:14px;
+    line-height:29px ;
+    letter-spacing: 0.7px;
+    margin-top: 20px;
+}
+.buton1 {
+    background: #55bba6;
+    display: block;
+    color:white;  
+    border: 2px solid #55bba6 ;
+    padding: 14px 40px;
+    letter-spacing: 0.7px;
+    border-radius: 0px;    
+    cursor: pointer;
+    text-align: center;
+    font-family: Sofia Pro;
+    font-size:20px;
+    width: 230px;
+    height: 50px;   
+}
+.buton2 {
+    background: rgba(255, 255, 255, 0.2);  
```

Comment:
incearca sa transformi din rgba in hex, ideal ar fi sa folosesti peste tot doar hex